src/app/content/highlights/components/utils/showConfirmation.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MessageProvider from '../../../../MessageProvider';
+import { assertDocument, assertNotNull } from '../../../../utils';
+import ConfirmationModal from '../ConfirmationModal';
+
+export default async() => {
+  const document = assertDocument();
+  const domNode = document.createElement('div');
+
+  domNode.id = `dialog-${Math.random().toString(36).substring(7)}`;
```

Comment:
the uuid library is already a dependency through the highlighter, you can add it to use here